rules-tests/Php80/Rector/Class_/StringableForToStringRector/Fixture/extends_with_interface.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\Class_\StringableForToStringRector\Fixture;
+
+use Rector\Tests\Php80\Rector\Class_\StringableForToStringRector\Source\ParentInterfaceWithStringable;
+
+final class ExtendsWithInterface implements ParentInterfaceWithStringable
+{
+    public function __toString()
+    {
+        return 'I can stringz';
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php80\Rector\Class_\StringableForToStringRector\Fixture;
+
+use Rector\Tests\Php80\Rector\Class_\StringableForToStringRector\Source\ParentInterfaceWithStringable;
+
+final class ExtendsWithInterface implements ParentInterfaceWithStringable
+{
+    public function __toString(): string
+    {
+        return 'I can stringz';
+    }
+}
```

Comment:
when parent interface already extends Stringable, it add return type: `string`, without `re-adding` Stringable into implements 👍

see https://3v4l.org/vgKoK